README.md

```diff
@@ -106,12 +108,16 @@ You can see which authorities have the feature enabled at the Authorities admin
 
 #### Adding councillors for an authority
```

Comment:
The current structure of this section makes it a bit hard to follow as a check list, which is how use use it as you work through the steps.

I think front loading the key actions in each par, and even using bold emphasis to highlight the steps could help.

I'll give that a go.